Configuration/AlCa/python/autoCond.py

```diff
@@ -54,19 +54,19 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for full Phase1 2018 (and 0,0,0-centred beamspot)
     'phase1_2018_design'       :  '106X_upgrade2018_design_v5',
```

Comment:
shouldn't this be updated as well?
https://cms-conddb-prod.cern.ch/cmsDbBrowser/list/Prod/gts/106X_upgrade2018_design_v5 contains 2017ULV2 payloads instead of the new 2018ULV1 appropriate for 2018.

Reply:
so this is in a funny state already. It uses the 0.08/0.3 PF rec-hit thresholds and not the ones for which this regression is appropriate. We should figure out what to do here and if ecal updates the PF rec-hit thresholds

Reply:
The design and cosmics scenarios have not been a target of the UL and so have not been updated except on subsystem request. For the cosmics scenarios, AlCa could create the GTs ourselves, as there is a fixed tag difference between the collision and cosmic tags. But for the design scenarios AlCa would need subsystem input.

@kpedro88 and @fabiocos Would updates to GTs for design and cosmics scenarios be allowed in 10_6_X? Would dedicated relval campaigns in 11_0_X be required prior to merging changes to these GTs?